catalog/spatial/registry/registry-federation-admin-service-impl/src/main/java/org/codice/ddf/registry/federationadmin/service/impl/FederationAdminServiceImpl.java

```diff
@@ -626,7 +626,10 @@ private String stringifyProcessingErrors(Set<ProcessingDetails> details){
         for(ProcessingDetails detail : details){
             pw.append(detail.getSourceId());
             pw.append(":");
-            detail.getException().printStackTrace(pw);
+            if(detail.hasException()) {
+                detail.getException()
+                        .printStackTrace(pw);
```

Comment:
We want to avoid printing stack traces to system out but this is just to a string so we don't lose it when creating a new exception.